src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/swift/SwiftAPIClient.java

```diff
@@ -65,7 +72,7 @@ private boolean checkProperties(Properties properties) {
         return SapsPropertiesUtil.checkProperties(properties, propertiesSet);
     }
 
-    public void createContainer(String containerName) {
```

Comment:
Return with the `public` term for encapsulating this method for now to maintain the default in the code.

Reply:
The default privacy mode is sufficient for this method.

Reply:
generally speaking: making the class declaration public. try to define an "API" for the class. methods in this API should be public. the others should be private (unless they are required for testing, then protected). There is also a friendly mode but ... ignore it.